sdk/servicebus/service-bus/src/util/errors.ts

```diff
@@ -137,7 +102,7 @@ export function getReceiverClosedErrorMsg(
   if (sessionId == undefined) {
     return (
       `The receiver for "${entityPath}" has been closed and can no longer be used. ` +
-      `Please create a new receiver using the "createReceiver" function on the ${clientType}.`
+      `Please create a new receiver using the "createReceiver" function on the ServiceBusClient.`
```

Comment:
getReceiver

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/8173/commits/7eeec6e0f8e321f5cf43d79c49c391d698e5aa2b